src/utils/json.ts

```diff
@@ -0,0 +1,8 @@
+export function tryParseJson(input: string): unknown | null {
+  try {
+    return JSON.parse(input);
+  } catch (e) {
+    console.warn('unable to parse JSON', e);
```

Comment:
Use logger. Atm logger just routes to console but that may not always be the case. I thought there was a lint rule enabled to prevent console but I guess it's not enabled in this project. Bonus points for fixing that :)